### DISCUSSION TITLE
Using wg-easy on Docker Desktop Windows

### DISCUSSION THREAD

**Discussion by DobleJSoto** [wg-easy/wg-easy#506] [Q&A]
Hi, I'm trying to deploy wg-easy in Windows Docker, and everything seems okay. However, I cannot connect between other peers as there is no response to ping or RDP

**Comment by Pleonazm** [comment]
Docker on windows (especially if it uses WSL) has some layers of isolation for security reasons.
Difficult to say what may be a problem here, you may try this at the beginning
Check if there is a connection from Docker Host to the public IP of clients.
Check networking host mode, to eliminate docker-networking problems

**Comment by hvmzx** [comment]
> Hi, I'm trying to deploy wg-easy in Windows Docker, and everything seems okay. However, I cannot connect between other peers as there is no response to ping or RDP

Did you end up making it work?
